src/components/Timer.jsx

```diff
@@ -1,22 +1,30 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
-import moment from 'moment';
+import { formatDuration } from '../utils';
+import { updateDuration } from '../actions';
 
-export const Timer = () => {
-  const [duration, setDuration] = useState(0)
+const TimerPure = ({ duration, updateDuration}) => {
   useEffect(() => {
     setTimeout(() => {
-      setDuration(duration + 1);
+      updateDuration();
```

Comment:
Spoko zostaw tak, ale performance apki strasznie stracił na tej akcji ;) ale to napewno nie zauważalne przy tak małej apce